takealot.user.js

```diff
@@ -0,0 +1,563 @@
+// ==UserScript==
+// @name           Import Takealot releases to MusicBrainz
+// @description    Add a button to import Takealot releases to MusicBrainz
+// @version        2016.03.09.2
+// @namespace      http://lenjoubertblog.blogspot.com
```

Comment:
Since it is meant to be merged into murdos/userscripts, you should use "https://github.com/murdos/musicbrainz-userscripts" as namespace imho.